### ISSUE TITLE
[Feature request] Pass custom `fetch` function

### ISSUE THREAD

**Issue by KaKi87** [center-key/fetch-json#32]
Hello,

It would be nice to be able to pass a specific `fetch` function to wrap with `ofetch`, so that it could be used in peculiar environments such as [SvelteKit's SSR `load` function](https://kit.svelte.dev/docs/load#making-fetch-requests).

Thanks

**Comment by dpilafian** [comment]
Potentially relevant SO answer:

**Setting a global header with ofetch in nuxt 3**
https://stackoverflow.com/a/75605263

**Comment by KaKi87** [comment]
No, the answer you attached is about making the wrapper (whether it is `fetch-json` or `ofetch`) globally available, which isn't what I'm asking for.

What I'm asking for is the ability to **pass** a `fetch` function as **parameter** to `fetchJson` so that it uses the specified one instead of `window.fetch`.

Thanks
